fetch-api/js/app.js

```diff
@@ -0,0 +1,70 @@
+const select = document.getElementById('breeds');
+const card = document.querySelector('.card');
+
+select.addEventListener('change', fetchBreedImage);
+card.addEventListener('click', fetchBreedImage);
+document.querySelector('form').addEventListener('submit', postInputFieldData);
+
+Promise.all([
+    fetchData('https://dog.ceo/api/breeds/image/random'),
+    fetchData('https://dog.ceo/api/breeds/list')
+])
+    .then(data => {
+        generateImage(data[0].message);
+        generateBreedOptions(data[1].message);
+    });
+
+//Functions:
+function checkStatus(response) {
```

Comment:
What about `jsdoc` for describing the function?
I thought you were using this.
I would start getting into that habit now.

Reply:
Added jsdoc to all functions.